src/steps/command.ts

```diff
@@ -238,7 +247,7 @@ export class CommandStep extends LabeledStep {
         /* eslint-disable @typescript-eslint/camelcase */
         return {
             ...(await super.toJson(opts)),
-            command: transformCommand(this.command),
+            command: Command[transformCommandKey](this.command),
```

Comment:
ah, hold on, it's not static anymore...

Reply:
This seems to work:
```ts
const sym: unique symbol = Symbol('x');
class X {
 private static [sym] = 'z';

 public y() {
     console.log(X[sym])
 }
}

const x = new X();
x.y();
```
https://www.typescriptlang.org/play?ts=4.3.4#code/MYewdgzgLgBBCeBbAXDArmAlgRzQUziQCMQAbGAXhgGViyAKAcgA9GBKAbgChhSBDCBBgANGAG8uMAA4AnTADc+UAtCWZgMANoJEAXUoxGAL0bdJUtEVLqY8em3GSYzmKEhk8AOlIgA5vWFtJF02SQBfLgiecGgYZgMwPAB3EXtuZk87TiA